python/cudf/cudf/tests/test_doctests.py

```diff
@@ -0,0 +1,85 @@
+import doctest
+import inspect
+import os
+
+import numpy as np
+import pytest
+
+import cudf
+
+
+def _name_in_all(parent, name):
+    return name in getattr(parent, "__all__", [])
+
+
+def _is_public_name(parent, name):
+    return not name.startswith("_")
+
+
+def _find_doctests_in_obj(finder, obj, criteria):
+    """Find all doctests in an object.
+
+    Args:
```

Comment:
Sneaking in a Google docstring into a numpydoc codebase. I see you 👁️

Reply:
Aw dang it! My bad. I'll change that.